homeassistant/components/switchbee/strings.json

```
@@ -0,0 +1,31 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "Setup SwitchBee integration with Home Assistant.",
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]"
```

```suggestion
          "password": "[%key:common::config_flow::data::password%]",
          "switch_as_light": "Initialize switches as light entities"
```
Missing:
https://github.com/home-assistant/core/blob/3d5fcbe634902ae6795b57784288d2f26b2b64a6/homeassistant/components/switchbee/translations/en.json#L17